arangod/Utils/Events.h

```diff
@@ -58,8 +58,10 @@ void TruncateCollection(std::string const& db, std::string const& name,
                         OperationResult const& result);
 void CreateDatabase(std::string const& name, Result const& result, ExecContext const& context);
 void DropDatabase(std::string const& name, Result const& result, ExecContext const& context);
-void CreateIndex(std::string const& db, std::string const& col,
-                 VPackSlice const& slice, ErrorCode result);
+// index creation start
+void CreateIndex(std::string const& db, std::string const& col, VPackSlice slice);
+// index creation result
+void CreateIndex(std::string const& db, std::string const& col, VPackSlice slice, ErrorCode result);
```

Comment:
Please rename these to make it more clear, e.g., `CreateIndexBegin` and `CreateIndexEnd` or something like that.